packages/lab/src/dialog/Dialog.tsx

```diff
@@ -28,6 +28,20 @@ import { useComponentCssInjection } from "@salt-ds/styles";
 import dialogCss from "./Dialog.css";
 import { DialogContext } from "./DialogContext";
 
+interface ConditionalWrapperProps {
+  condition: boolean;
+  wrapper: unknown;
```

Comment:
I'm a bit unsure about how to type the wrapper here - if anyone has any suggestions